crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -390,6 +390,12 @@ impl CheckpointStore {
                 .computation_cost,
         })
     }
+
+    pub fn checkpoint_db(&self, path: &Path) -> SuiResult {
+        self.checkpoint_content
```

Comment:
Is the plan to add more tables latter?

Reply:
same here, it is already checkpointing the entire db.